README.md

```diff
@@ -23,22 +23,21 @@ Heroic is built with Web Technologies like: TypeScript, React, NodeJS and Electr
 
 ## How to use it
 
-- [Download and Install](#installation) the package for your distro or the universal AppImage file on the Releases Page;
+- [Download and Install](#installation) the package for your distro, the universal AppImage for Linux or the exe for Windows on the Releases Page;
 - If you used Legendary before, it loads your library and installed games. If not, it will ask you to login first. Just follow the instructions.
 
 ## Feature available right now
 
 - Login with an existing Epic Games account
 - Install/Uninstall Games
 - Import an already installed game
-- Play online (EAC not supported)
+- Play online (EAC not supported on Linux)
 - Update installed Games
 - Repair installed Games
 - Move installed games to different folders
-- Multiple downloads at the same time
-- Play games using the default wine and default prefix
-- Play game with custom wine (Lutris Wine/Proton maybe but can lead to bugs)
-- Run games on custom wine prefix
+- Play games using the default wine and default prefix (Linux)
+- Play game with custom wine (Lutris Wine/Proton maybe but can lead to bugs) (Linux)
```

Comment:
Ik this was here before but wouldnt "This might lead to bugs" instead of "Lutris Wine/Proton maybe but can lead to bugs" fit better

Id have the line like this:
- Play game with custom wine installs (This might lead to bugs) [Linux]